### PR TITLE
created User class

### PR DESCRIPTION
**Pull request by anahittum**
created User class

### CODE REVIEW

File: src/homeworks/week2/user/creation/User.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public static void createUsers(User[] userList) {
+        for (int i = 0; i < userList.length; i++) {
+            userList[i] = new User(new Scanner(System.in).nextLine(), new Scanner(System.in).nextInt());
+        }
+    }
+
+    public static void main(String[] args) {
+        User[] userList = new User[3];
+
+        for (int i = 0; i < userList.length; i++) {
```
Comment (gevorgyanarman) [inline src/homeworks/week2/user/creation/User.java:50]: this will create 3x3 users

Comment (gevorgyanarman) [inline src/homeworks/week2/user/creation/User.java:50]: there is no need to create loop here

File: src/homeworks/week2/user/creation/User.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    public static void wait1Second() {
+        long start = System.currentTimeMillis();
+        while (System.currentTimeMillis() - start < 1000) {
+
+        }
+    }
+
+    public static void createUsers(User[] userList) {
```
Comment (gevorgyanarman) [inline src/homeworks/week2/user/creation/User.java:41]: wouldn't it be better if createUsers took nothing and returned User[]?

File: src/homeworks/week2/user/creation/User.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Override
+    public String toString() {
+        return "The name is: " + getName() + ". " + getName() + "'s height is: " + getHeight();
+    }
+
+    private static void printUsers(User[] users) {
+        for (int i = 0; i < users.length; i++) {
+            System.out.println(users[i]);
+            wait1Second();
```
Comment (gevorgyanarman) [inline src/homeworks/week2/user/creation/User.java:30]: there was no requirement to wait

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week2/user/creation/User.java (hunk 3 above)
Comment (anahittum) [reply src/homeworks/week2/user/creation/User.java:30]: Seems that "print all users one by one" implies that.

File: src/homeworks/week2/user/creation/User.java (hunk 2 above)
Comment (anahittum) [reply src/homeworks/week2/user/creation/User.java:41]: That's how I did it initially but then printing the users was more complicated so I modified it to this.

File: src/homeworks/week2/user/creation/User.java (hunk 1 above)
Comment (anahittum) [reply src/homeworks/week2/user/creation/User.java]: Corrected.

File: src/homeworks/week2/user/creation/User.java (hunk 2 above)
Comment (gevorgyanarman) [reply src/homeworks/week2/user/creation/User.java:41]: Although it works it is better to pass nothing but to get an array

File: src/homeworks/week2/user/creation/User.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    public static void createUsers(User[] userList) {
+        for (int i = 0; i < userList.length; i++) {
+            userList[i] = new User(new Scanner(System.in).nextLine(), new Scanner(System.in).nextInt());
+        }
+    }
+
+    public static void main(String[] args) {
+        User[] userList = new User[3];
+
+            createUsers(userList);
```
Comment (gevorgyanarman) [inline src/homeworks/week2/user/creation/User.java:50]: please remove the extra spaces

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week2/user/creation/User.java (hunk 4 above)
Comment (anahittum) [reply src/homeworks/week2/user/creation/User.java]: Done.

File: src/homeworks/week2/user/creation/User.java (hunk 2 above)
Comment (anahittum) [reply src/homeworks/week2/user/creation/User.java]: Done.

**Pull request merged**